Update email-logs.md with info about DeliveryStatus possible values (#124092)
Azure/azure-content: articles/communication-services/concepts/analytics/logs/email-logs.md

## Email Status Update operational logs

*Email status update operational logs* provide in-depth insights into message-level and recipient-level delivery status updates on your sendmail API requests.

> - Message-level status updates provide the status of the long running email send operation (similar to the status updates you receive through calling our GET APIs). These are marked by the absence of `RecipientId` property since these update are for the entire message and not applicable to a specific recipient in that message request. `DeliveryStatus` property contains the message-level delivery status. Possible values for `DeliveryStatus` for this type of event are: `Dropped`, `OutForDelivery`, `Queued`.

Reviewer: ```suggestion
- Message-level status updates provide the status of the long-running email send operation (similar to the status updates you receive through calling our GET APIs). These are marked by the absence of `RecipientId` property because these updates are for the entire message and not applicable to a specific recipient in that message request. `DeliveryStatus` property contains the message-level delivery status. Possible values for `DeliveryStatus` for this type of event are `Dropped`, `OutForDelivery`, and `Queued`.
```

> - Recipient-level status updates provide the status of email delivery for each individual recipient to whom the email was sent in a single message. These contain a `RecipientId` property with the recipient's email address. Recipient-level delivery status is provided in the `DeliveryStatus` property. Possible values for `DeliveryStatus` for this type of event are: `Delivered`, `Expanded`, `Failed`, `Quarantined`, `FilteredSpam`, `Suppressed`, `Bounced`.

Reviewer: ```suggestion
- Recipient-level status updates provide the status of email delivery for each individual recipient to whom the email was sent in a single message. These contain a `RecipientId` property with the recipient's email address. Recipient-level delivery status is provided in the `DeliveryStatus` property. Possible values for `DeliveryStatus` for this type of event are `Delivered`, `Expanded`, `Failed`, `Quarantined`, `FilteredSpam`, `Suppressed`, and `Bounced`.
```
